### PR TITLE
Fix vulnerabilities ALAS-2020-1466 ALAS-2020-1445

### PR DESCRIPTION
**Pull request by yunwen-wang**
Signed-off-by: Amit Chaudhari <achaudhari@guidewire.com>

**Is this a bug fix or adding new feature?**
Fix security vulnerabilities:
- [ALAS-2020-1466](https://alas.aws.amazon.com/AL2/ALAS-2020-1466.html)
- [ALAS-2020-1445](https://alas.aws.amazon.com/AL2/ALAS-2020-1445.html)

**What is this PR about? / Why do we need it?**
As mentioned above, fixing security vulnerabilities.

**What testing is done?** 
We verified that after the changes, the high level security vulnerabilities are resolved.

### CODE REVIEW

**Comment by jbhennin** [comment]
What is the status of this PR?

**Comment by yunwen-wang** [comment]
I signed it

**Comment by yunwen-wang** [comment]
@jbhennin Sorry for the delayed reply. Looks like I don't have CLA signature. I just signed CLA and hope the scan result will update.

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,8 @@ RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-efs-csi-driver
 
 FROM amazonlinux:2.0.20200602.0
```
Comment (wongma7) [inline Dockerfile:34]: Should we use latest tag, would it have the newest versions of these packages?

**Comment by yunwen-wang** [comment]
@wongma7 I made an empty push to try to trigger another scan for CLA. But looks like I still didn't pass the CLA check. I'm wondering what's the best way you would recommend to re-trigger the CLA scan? Thanks!

**Comment by wongma7** [comment]
commenting /check-cla should do it. Make sure the email of your commit (currently achaudhari guidewire com> matches the email you signed the cla with

**Comment by yunwen-wang** [comment]
I signed it

**Comment by yunwen-wang** [comment]
@wongma7 Hi, I successfully passed the CLA scan. The `pull-aws-efs-csi-driver-e2e` test failed previous times but seems to pass in the beginning. Could this be an intermediate issue? Many thanks 🙏

File: Dockerfile (hunk 1 above)
Comment (amitkc1) [reply Dockerfile:34]: We can push the change for `amazonlinux:2.0.20210219.0`. This tag does have the packages to be addressed in the vulnerabilities. We don't know if the latest tag will introduce any other vulnerabilities. Is that okay ?

Comment (wongma7) [reply Dockerfile:34]: @amitkc1  yep, I'll investigate using latest tag separately: https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/353

There are apaprently some other CVEs and I want to squash as many as possible but, as you say, we don't know if latest will introduce more. https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/343

**Comment by wongma7** [comment]
/lgtm

please squash if you can!

Sorry, our CI is quite unstable as we prepare for the next release!!!

**Comment by yunwen-wang** [comment]
@wongma7 Sure, just squashed it.

**Pull request merged**